tools/Cargo.toml

```diff
@@ -17,6 +17,7 @@ base64.workspace = true
 clap.workspace = true
 derivative.workspace = true
 fixed = { version = "1.26", optional = true }
+janus_aggregator_core.workspace = true
```

Comment:
This import appears to be unused now. Plus, it would prevent us from publishing the hpke_keygen/dap_decode/collect tools to crates.io in the future.